test/token/ERC20/CappedToken.test.js

```diff
@@ -7,10 +8,18 @@ const CappedToken = artifacts.require('CappedToken');
 contract('Capped', function ([_, owner, ...otherAccounts]) {
   const cap = ether(1000);
 
-  beforeEach(async function () {
-    this.token = await CappedToken.new(cap, { from: owner });
+  it('requires a non-null cap', async function () {
```

Comment:
~I think I prefer the word "invalid" here.~ (jk)

Reply:
Should be `non-zero` though :stuck_out_tongue: